packages/react-component-library/src/components/DataGrid/partials/StyledRow.tsx

```diff
@@ -4,11 +4,15 @@ import { selectors } from '@royalnavy/design-tokens'
 interface StyledRowProps {
   $hasHover?: boolean
   $hasFocus?: boolean
+  $depth?: number
+  $isLastInBranch?: boolean
 }
 
 const { color } = selectors
```

Comment:
We can use new exports now rather than restructuring

Reply:
Updated across the whole codebase with a bash script.